src/packages/core/components/input-color/input-color.element.ts

```diff
@@ -48,7 +51,11 @@ export class UmbInputColorElement extends UUIFormControlMixin(UmbLitElement, '')
 		return map(
 			this.swatches,
 			(swatch) => html`
-				<uui-color-swatch label=${swatch.label} value=${swatch.value} .showLabel=${this.showLabels}></uui-color-swatch>
+				<uui-color-swatch
+					?readonly=${this.readonly}
```

Comment:
@iOvergaard does it need to set this as `readonly` on `uui-color-swatches` explicit set `readonly` on `uui-color-swatch`?
https://github.com/umbraco/Umbraco.UI/pull/932/files#diff-8090b44f925ed46b626e45b3d05f30d71d493772a9ddf02effbb81f86c088475R99

Reply:
It doesn't set `disabled` attribute here either ;)

Reply:
@bjarnef Good point, but it doesn't look like it's hurting anything setting disabled explicitly. Would you like to make a PR with the disabled attribute?

Reply:
I guess `_handleSlotChange` may execute after? If `<uui-color-swatch>` e.g. inside `<uui-color-swatches>` is set to disabled or readonly, but `<uui-colors-swatches>` isn't, I guess the swatch is still disabled/readonly as it only explicit handle `disbled` and `readonly`.

I guess we may need an `else` to remove attribute as well in `<uui-colors-swatches>`?

https://github.com/umbraco/Umbraco.UI/pull/932/files#diff-8090b44f925ed46b626e45b3d05f30d71d493772a9ddf02effbb81f86c088475R91-R101